### PR TITLE
Refactor/pet details display

### PR DESCRIPTION
**Pull request by apete12**
What I did: 
- Refactor PetDetails component in order for id data type from useParams to be a number. 

Testing Notes/Extras: 

Did this break anything?
- [X]  No
- [ ]  Yes

Type of change
- [ ]  New feature (non-breaking change which adds functionality)
- [X]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Added testing for: 
- [ ]  Skip all the other stuff and briefly explain the fix.

Checklist:
- [X]  If this code needs to be tested, all tests are passing
- [X]  The code runs locally
- [X]  The code is DRY. If it's not, I tried my best
- [X]  I reviewed my code before pushing

What's next?
Check cypress testing for PetDetails.

### CODE REVIEW

File: src/components/PetDetails/PetDetails.js [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
 import './PetDetails.css'
 import { Link, useParams } from 'react-router-dom'
 import cat from '../../images/cat.png'
 
 function PetDetails({allPets}) {
   const { id } = useParams()
 
-  const foundPet = allPets.find(pet => pet.id === id)
-
+const foundPet = allPets.find(pet => pet.id === Number(id))
```
Comment (judy0ye) [inline src/components/PetDetails/PetDetails.js:8]: Nice catch of converting the string to a number!

**Review by judy0ye — APPROVED** [review]

**Review by aliceabarca — APPROVED** [review]
Awesome job converting the string into a number!

**Review by paulina-isabel — APPROVED** [review]
Great job checking the url paths.

File: src/components/Forms/Form.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const Form = ({ addNewPet, setLoading, setError }) => {
     e.preventDefault()
 
     const newPet = {
-      id: Date.now(),
+      id:  Date.now(),
```
Comment (nicktu12) [inline src/components/Forms/Form.js:19]: I think we already talked about this, but typically IDs are generated on the BE

**Pull request merged**